build/olm-catalog/template.yaml

```diff
@@ -30,6 +30,24 @@ spec:
     The operator adds support for management of multiple Ember CSI instances
     within a cluster and simplifies storage backend deployments.
 
+    ### Supported drivers
+    Ember-CSI includes a good number of storage drivers, but due to limitation on hardware availability only a small number of them have been validated at one point or another. In alphabetical order they are:
+
+    HPE3PARFC
```

Comment:
These all appear as a single line, should we make them a list by adding - to each one?